avocado/utils/asset.py

```diff
@@ -406,3 +390,49 @@ def get_metadata(self):
             with open(metadata_file, "r") as f:
                 metadata = json.load(f)
                 return metadata
+
+    @property
+    def asset_name(self):
+        return os.path.basename(self.parsed_name.path)
+
+    @property
+    def name_scheme(self):
+        """This property will return the scheme part of the name if is an URL.
+
+        Otherwise, will return None.
+        """
+        parsed = self.parsed_name
+        if parsed:
```

Comment:
In most parts of Avocado's code, we are using `if var is/is not None:`. I know there was a long discussion about it some time ago. I would like us to choose one and follow that as a standard.

Reply:
Not sure if have to choose, because they can be used in different situations. Sometimes makes sense to check if is not None, sometimes I need to run some basic checks as I explained on that long thread.